src/loaders/DesktopArticleVideoLoader.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import JWDesktopArticleVideoPlayer from 'jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer';
+import { DesktopArticleVideoLoaderProps } from 'loaders/types';
+import { jwPlayerExperimentTracker } from 'jwplayer/utils/videoTracking';
+import { checkIfUserInNoVideoExperiment } from 'utils/experiments/experiments';
+
+const DesktopArticleVideoLoader: React.FC<DesktopArticleVideoLoaderProps> = ({ videoDetails }) => {
+	// Disable video load if:
+	if (checkIfUserInNoVideoExperiment()) {
+		jwPlayerExperimentTracker.singleTrack('no-video-experiment');
```

Comment:
I'm wondering if we need to differentiate mobile vs desktop in the name here. This might need to be 'desktop-no-video-experiment' and 'mobile-no-video-experiment', unless we're sending extra data in this tracking to differentiate between these.